apm-collector/apm-collector-stream/src/main/java/org/skywalking/apm/collector/stream/worker/util/Const.java

```diff
@@ -1,13 +1,11 @@
-package org.skywalking.apm.collector.agentstream.worker;
+package org.skywalking.apm.collector.stream.worker.util;
 
 /**
  * @author pengys5
  */
 public class Const {
     public static final String ID_SPLIT = "..-..";
     public static final String IDS_SPLIT = "\\.\\.-\\.\\.";
```

Comment:
Miss these two constants in prev review. Through these, I can tell you use some regex split, it is **NOT** efficient at all. As you only want to split by a single char, I recommend you to provide a stream-mode String split util, instead of this.